kahuna/public/js/edits/image-editor.html

```diff
@@ -89,123 +89,149 @@
     </div>
 
     <div class="result-editor__editor">
-        <div class="result-editor__field-container image-info__wrap">
-            <span class="result-editor__field-label flex-no-shrink text-small">
-                Rights & restrictions
-            </span>
-
-            <div class="result-editor__usage-rights-container">
-                <div ng:hide="ctrl.showUsageRights" class="result-editor__field-value">
-                    <span>{{ctrl.usageRightsCategory || 'None'}}</span>
-                    <button class="image-info__edit" ng:click="ctrl.showUsageRights = !ctrl.showUsageRights">
-                        <gr-icon>edit</gr-icon>
+        <section>
+            <h1>Image Rights</h1>
+            <div class="result-editor__field-container image-info__wrap">
+                <span class="result-editor__field-label flex-no-shrink text-small">
+                    Rights & restrictions
```

Comment:
do we still use `&amp;` in HTML? I'm out of the loop :D

Reply:
Yep! oops!